src/database.js

```diff
@@ -524,6 +613,121 @@ Database.prototype.getSchema = function(callback) {
   );
 };
 
+/**
+ * Options object for listing sessions.
+ *
+ * @typedef {object} GetSessionsRequest
+ * @property {boolean} [autoPaginate=true] Have pagination handled
+ *     automatically.
+ * @property {string} [filter] An expression for filtering the results of the
+ *     request. Filter rules are case insensitive. The fields eligible for
+ *     filtering are:
+ *     - **`name`**
+ *     - **`display_name`**
+ *     - **`labels.key`** where key is the name of a label
+ *
+ *     Some examples of using filters are:
+ *     - **`name:*`** The instance has a name.
+ *     - **`name:Howl`** The instance's name is howl.
+ *     - **`labels.env:*`** The instance has the label env.
+ *     - **`labels.env:dev`** The instance's label env has the value dev.
+ *     - **`name:howl labels.env:dev`** The instance's name is howl and it has
+ *       the label env with value dev.
+ * @property {number} [maxApiCalls] Maximum number of API calls to make.
+ * @property {number} [maxResults] Maximum number of items to return.
+ * @property {number} [pageSize] Maximum number of results per page.
+ * @property {string} [pageToken] A previously-returned page token
+ *     representing part of the larger set of results to view.
+ */
+/**
+ * @typedef {array} GetSessionsResponse
+ * @property {Session[]} 0 Array of {@link Session} instances.
+ * @property {object} 1 The full API response.
```

Comment:
I don't think we get an apiResponse from GAPIC.

Reply:
See https://github.com/googleapis/nodejs-spanner/pull/132#discussion_r171342654